resources/qml/MessageView.qml

```diff
@@ -249,12 +249,12 @@ ScrollView {
             id: sectionHeader
 
             Column {
-                topPadding: 4
-                bottomPadding: 4
+                topPadding: userName_.visible? 4: 0
+                bottomPadding: Settings.bubbles? 0 : 3
```

Comment:
Imo that is a bit low with the bubble layout?

![image](https://user-images.githubusercontent.com/16312324/153773877-08ca9ee0-96f1-4905-83c0-93db94757b7b.png)